### DISCUSSION TITLE
How to to pull the env (Vercel declared Environment Variables) on Deployment Build?

### DISCUSSION THREAD

**Discussion by mytabworks** [vercel/next.js#58251] [Help]
### Summary

I am using drizzle on the script "build" I am running "next build && npx drizzle-kit push:mysql" but the command for "npx drizzle-kit push:mysql" is looking for my .env file to get the credentials for my DB, however all my declared Environment Variables are all stored on Vercel? how could I pull/generate the .env before it runs the the "build" script?

**Comment by icyJoseph** [comment]
I think, if you are using the Vercel CLI, you can do `vercel env pull --environment=preview` or production, as you need.

- https://vercel.com/docs/cli/env
